### PR TITLE
added basic auth error page and update auth logic to show on failed l…

### PR DESCRIPTION
**Pull request by meyerjrr**
If the logging in user is not found in Granteds database previously would cause a infinite loop in the frontend.
This fix will display a helpful error page if this occurs

### CODE REVIEW

File: web/src/utils/context/userContext.tsx [hunk 1]
```diff
@@ -37,33 +40,36 @@ const UserProvider: React.FC<Props> = ({ children }) => {
 
     // await Auth.currentSession();
 
-    const me = await getMe();
+    const me = await getMe().then((user) => {
```
Comment (JoshuaWilkes) [inline web/src/utils/context/userContext.tsx]: Why the change here?
this effectively means that me is never defined?

Using await + .then doesn't make sense

File: web/src/utils/context/userContext.tsx [hunk 2]
```diff
@@ -37,33 +40,36 @@ const UserProvider: React.FC<Props> = ({ children }) => {
...
     switch (event) {
       case "signIn":
+        setamplifyLoggedIn(true);
+        setamplifyUser(data);
       case "cognitoHostedUI":
-        await getUser().then(() => {
+        await getUser().then((out) => {
           if (user !== undefined) {
-            setLoading(false);
+            //setLoading(false );
```
Comment (JoshuaWilkes) [inline web/src/utils/context/userContext.tsx]: is this commented out?
Seems like it would still be needed if the call to getMe fails?

**Review by JoshuaWilkes — CHANGES_REQUESTED** [review]
Some comments to clarify

File: web/src/utils/context/userContext.tsx (hunk 1 above)
Comment (meyerjrr) [reply web/src/utils/context/userContext.tsx]: Good catch, was testing a race condition. Reverted it

File: web/src/utils/context/userContext.tsx (hunk 2 above)
Comment (meyerjrr) [reply web/src/utils/context/userContext.tsx]: It called the same thing in the useEffect, so was effectively not needed. Can just be deleted I think

Comment (JoshuaWilkes) [reply web/src/utils/context/userContext.tsx]: ok

File: web/src/utils/context/userContext.tsx [hunk 3]
```diff
@@ -38,32 +41,28 @@ const UserProvider: React.FC<Props> = ({ children }) => {
     // await Auth.currentSession();
 
     const me = await getMe();
-    console.debug({ msg: "getMe response", me });
+    console.log(me);
```
Comment (JoshuaWilkes) [inline web/src/utils/context/userContext.tsx]:
```suggestion
```

**Review by JoshuaWilkes — CHANGES_REQUESTED** [review]
Looks like tests failing in a strange way, unrelated to this PR. Did we merge something in that caused this?

**Review by JoshuaWilkes — APPROVED** [review]
